Source/WebCore/platform/graphics/gstreamer/MediaPlayerPrivateGStreamer.cpp

```diff
@@ -1797,14 +1797,38 @@ void MediaPlayerPrivateGStreamer::handleMessage(GstMessage* message)
         asyncStateChangeDone();
         break;
     case GST_MESSAGE_STATE_CHANGED: {
-        if (!messageSourceIsPlaybin || m_isDelayingLoad)
-            break;
-
         GstState newState;
         gst_message_parse_state_changed(message, &currentState, &newState, nullptr);
-
         GST_DEBUG_OBJECT(pipeline(), "Changed state from %s to %s", gst_element_state_get_name(currentState), gst_element_state_get_name(newState));
```

Comment:
Move this to just before the dump code, as it was before, please.

Reply:
I don't follow. This code does not change. Should I remove the extra line removal?

Reply:
No, sorry. I was confused :)
One small issue with this change though is that we'd get "Changed state from..." logs for all elements of the pipeline now. Can you move this log to after the early `break`?